animations/implicit_widgets.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: page
+title: "Flutter implicit animations"
+permalink: /animations/implicit_widgets.html
+---
+
+* TOC Placeholder
+{:toc}
+
+
+If you’re new to Flutter or new to animation, consider starting with the built-in Flutter implicit widgets. Implicit animations are useful when you want to quickly add an animation without having to write a lot of animation code because the Flutter SDK implicit widgets already include the `AnimationController`, the ticker provider, the listener, and other animations elements.
+
+Implicit animations are distinct from explicit animations which involve building the animation action using `AnimationController`. And implicit animations are different from transition widgets since you don't need to create an animation widget using `AnimatedWidget`, you simply use an existing implicit animation widget such as `AnimatedOpacity` or `AnimatedIcon`.
```

Comment:
The second sentence conflates the experience of _creating_ a transition widget from _using_ one. Using a transition widget does not involve `AnimatedWidget`. I think this paragraph doesn't need to talk about how to create a custom transition widget. 

In addition, it's probably helpful to clarify that instead of using an `AnimationController` explicitly, an implicit animation simply responds to changes of a specific property and make such changes appear smoothly.